src/types.ts

```diff
@@ -56,3 +60,55 @@ export interface MapOptions {
 
   layers: LayerOptions[]
 }
+
```

Comment:
these seems over-specified. See comment on the .d.ts version of file.

Reply:
Simplified.